tests/migrations/test_operations.py

```diff
@@ -1,3 +1,5 @@
+from unittest import skipUnless
```

Comment:
The issue is not _operation_ specific and should probably live in the `schema` test app.